pkg/controller/kieapp/webconsole/webconsole.go

```diff
@@ -0,0 +1,102 @@
+package webconsole
+
+import (
+	"errors"
+	"fmt"
+	"github.com/ghodss/yaml"
+	"github.com/gobuffalo/packr/v2"
+	"github.com/kiegroup/kie-cloud-operator/pkg/controller/kieapp/logs"
+	creator "github.com/kiegroup/kie-cloud-operator/pkg/controller/kieapp/webconsole/creator"
+	"github.com/kiegroup/kie-cloud-operator/pkg/controller/kieapp/webconsole/factory"
+	"strings"
+)
+
+var log = logs.GetLogger("kieapp.webconsole")
+
+func LoadWebConsoleEnrichmentUI() {
+	files, err := loadFilesOnlyWithBox("consoleyamlsamples", "noop", "noop")
+	if err != nil {
+		log.Error(err, "yaml stings not loaded")
+	} else {
+		_, err = applyMultipleWebConsoleYamls(files)
+		if err != nil {
+			log.Error(err, "web console yamlsamples not successfully applied")
+		}
+	}
+	files, err = loadFilesOnlyWithBox("webconsole", "noop", "noop")
```

Comment:
This operator only needs sample yamls in this code. The only other thing is the link, but it won't be based on a yaml and will have to be separately figured out based on the deployed object, so please take out the other stuff from here and just focus on creating the yaml samples.

Reply:
Given the above, chance the function name to something like "CreateSampleYAMLs"

Reply:
modified.